chef/cookbooks/crowbar/templates/default/crowbar-post-upgrade.sh.erb

```diff
@@ -53,12 +53,13 @@ fi
 
 if [[ -f /sbin/drbdadm ]] ; then
     # Wait until DRBD is started
-    while crm resource status ms-drbd-{postgresql,rabbitmq} | grep "NOT running"; do
+    while [[ $(drbdadm status | wc -c) -eq 0 ]]; do
         sleep 1
```

Comment:
IMO we should log something here. Otherwise it will be hard to debug if for some reason a drbd is never started correctly. It should also probably timeout after a while.